egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -19,8 +19,10 @@ rand_prune=4.0 # Relates to a speedup we do for LDA.
 minibatch_size=512  # This default is suitable for GPU-based training.
                     # Set it to 128 for multi-threaded CPU-based training.
 
-samples_per_iter=400000 # each iteration of training, see this many samples
+frames_per_iter=80000 # each iteration of training, see this many samples
                         # per job.  This option is passed to get_egs.sh
+                        # this is 1/5 of that used for FF-DNN
+iters_per_archive=20  # number of iterations worth samples to store in an ark file
```

Comment:
Temporarily we will store egs for multiple iters in the same archive file. This is done to reduce the number of files created by get_egs.sh. get_egs.sh will later be modified to reduce the number of files created. This option will be removed then.